iceberg/iceberg-handler/src/test/queries/positive/iceberg_rename.q

```diff
@@ -0,0 +1,48 @@
+
+-- create an unpartitioned table
+create table iceorgin (id int, name string) Stored by Iceberg TBLPROPERTIES ('format-version'='2');
+
+-- insert some values
+insert into iceorgin values (1, 'ABC'),(2, 'CBS'),(3, null),(4, 'POPI'),(5, 'AQWR'),(6, 'POIU'),
+(9, null),(8,'POIKL'),(10, 'YUIO');
+
+-- do some deletes
+delete from iceorgin where id>9 OR id=8;
+
+select * from iceorgin order by id;
+
+-- do the rename
+
+alter table iceorgin rename to icerenamed;
+
+select * from icerenamed order by id;
+
+-- create a partitioned table
+create table iceorginpart (id int) partitioned by (part string) Stored by Iceberg TBLPROPERTIES ('format-version'='2');
```

Comment:
If i am understand correctly, i don't think only v2 table support rename operation. This qtest will mislead user that only v2 table can do rename operation, Actually, It shoud be have nothing to do with format-version. Should we also add v1 table qtest?
Please correct me if am wrong. Thx.

Reply:
Thanx @zhangbutao for the review, I have added a test for v1 as well